src/modules/Telemetry/Sensor/LTR390UVSensor.cpp

```diff
@@ -56,9 +59,11 @@ bool LTR390UVSensor::getMetrics(meshtastic_Telemetry *measurement)
             ltr390uv.setGain(
                 LTR390_GAIN_1); // x1 gain will already max out the sensor at direct sunlight, so no need to increase it
             ltr390uv.setMode(LTR390_MODE_ALS);
+
+            return true;
         }
     }
 
-    return true;
+    return false;
```

Comment:
can we set has_lux and has_v_lux to false here too? i think we're good after that change.